src/test/extensionDependencies.ts

```diff
@@ -0,0 +1,5 @@
+export const extensionDependencies = [
+  "pokey.parse-tree",
```

Comment:
I would be inclined to just use most recent, no?

Reply:
Yep yep, I was just thinking — how do I know that as a dev I'm running against an outdated version rather than the most recent, since we do an install script apart from the test runner. Not a huge deal though.

Reply:
Not sure I have a good answer here.  I'm not sure we want to rerun install script every time, because what if they're testing a local copy of parse tree extension

Reply:
@Will-Sommers this comment turned out to be prescient.  @AndreasArvidsson just ran into an issue where he was getting failing local tests due to an old parse tree extension version, and spent some time banging his head against it because there was no clear indication that was the cause of the problem.

Here is a proposed solution.  WDYT? https://github.com/cursorless-dev/cursorless/issues/860